files/image_config/monit/conf.d/sonic-host

```diff
@@ -65,3 +65,8 @@ check program arp_update_checker with path "/usr/bin/arp_update_checker" every 1
 check program controlPlaneDropCheck with path "/usr/bin/control_plane_drop_check"
     every 5 cycles
     if status != 0 for 3 cycle then alert repeat every 1 cycles
+
+# Periodically update oper status of mgmt interface in STATE_DB 
+check program mgmtOperStatus with path "/usr/bin/mgmt_oper_status.py"
+    every 1 cycles
+    if status != 0 for 3 cycle then alert repeat every 1 cycles
```

Comment:
have we tested out config relolad/minigraph  scenario. Are we not getting monit error  as that can impact nightly

Reply:
Verified reload/reboot, did not see the monit log message, is there any other specific concern on why this error might get logged?